lib/rules/require-presentational-children.js

```diff
@@ -93,6 +98,11 @@ export default class RequirePresentationalChildren extends Rule {
           return;
         }
 
+        // If node tag has an attribute in this list, we return
```

Comment:
Added a test that is failing so this is not the correct approach, will refactor.